src/Mailer/Message.php

```diff
@@ -396,6 +396,19 @@ public function getReplyTo(): array
         return $this->replyTo;
     }
 
+    /**
+     * Add "Reply-To" address.
+     *
+     * @param string|array $email Null to get, String with email,
```

Comment:
```suggestion
     * @param string|array $email String with email,
```
I don't think we do 'null to get' in add methods anymore.

Reply:
I copied the same comment of the 'addTo' method. You are right, this method will always return `$this` from the `addEmail` method (the same for addTo, AddBcc and addCc).
Should I change all the other 3 comments too?